crates/agglayer-config/src/tracing.rs

```diff
@@ -69,14 +77,15 @@ impl From<LogLevel> for EnvFilter {
 /// appropriate enum variant. If the string is not recognized to be either
 /// `stdout` or `stderr`, it is assumed to be a file path.
 #[derive(Serialize, Debug, Clone, Default, PartialEq, Eq)]
-pub enum LogOutput {
+pub enum TracingOutput {
     #[default]
     Stdout,
     Stderr,
     File(PathBuf),
+    Otlp,
```

Comment:
I'm not sure `Otlp` is an output format.
It means that if we're defining `Otlp` here, we do not have a way to define the `log` output?